tests/pyccel/scripts/print_sp_and_end.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+#  ------------------------string with sep and end ---------------------------
+
+print('ahoy', 'world', sep=' ')
```

Comment:
This may be beyond the scope of this PR. But it may also be a 1 line fix so I'm going to ask anyway:
Does the concatenation method not work for sep?
I.e:
```
(' // '+self._print(sep)+' // ').join(sub_strs)
```

Reply:
hh, I thought it was unnecessary to add sep and end for Fortran.
the method you provide works only for strings. presence of a non string variable will cause a problem.
anyway. I will add support to that. it is easy.

Reply:
> hh, I thought it was unnecessary to add sep and end for Fortran.
That's why I said it's beyond the scope of this PR
> the method you provide works only for strings. presence of a non string variable will cause a problem.
That's true